QUALITY_DECLARATION.md

```diff
@@ -80,11 +80,28 @@ There is an automated test which runs a linter that ensures each file has at lea
 
 `libyaml_vendor` is a package providing solely CMake files and therefore does not require API or feature tests and has no coverage or performance requirements (sections [4.i - 4.iv]).
 
+### Feature Testing [4.i]
+
+As we mentioned in the Quality Declaration of `libyaml`, ROS 2 only uses the API for reading YAML files. This will be the only `libyaml` API tested in the ROS 2 buildfarm.
+
+### Public API Testing [4.ii]
+
+As we mentioned in the Quality Declaration of `libyaml`, ROS 2 only uses the API for reading YAML files. This will be the only `libyaml` API tested in the ROS 2 buildfarm.
+
+### Coverage [4.iii]
+
+Lastest code coverage can be found [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/).
```

Comment:
I don't see any `libyaml` test results there yet. Is that because they will only show after this PR is merged? Where can I check the coverage with this PR?

Reply:
This PR includes the following line in the CMake which allows to check the coverage

```make
list(APPEND extra_cmake_args -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS})
```

I can split the PR in two: benchmark and QD.

Reply:
Finally I splitted in two PRs, if we merge this one first then we will able to see the coverage and benckmark in the buildfarm.